manifests/cpp.yml

```
@@ -1,35 +1,42 @@
 ---
+# For features that are not yet available in the latest release,  
+# please specify a version greater than the current release using `>`.  
+# DO NOT SPECIFY A NON EXISTING RELEASE.
+#
+# For example, if the latest release is `v1.0.0` and a feature like Baggage  
+# is only supported on the `main` branch and in the upcoming release,  
+# use `>v1.0.0` to indicate that requirement.
 tests/:
   parametric/:
     test_128_bit_traceids.py:
-      Test_128_Bit_Traceids: v1.0.1.dev
+      Test_128_Bit_Traceids: >v1.0.0
```

removes the `v`, and add quotes : 

```suggestion
      Test_128_Bit_Traceids: ">1.0.0"
```

Some for all others